### ISSUE TITLE
Specific audio generation

### ISSUE THREAD

**Issue by gcunhase** [fatchord/FFTNet#4]
Hi, is it possible to use this model conditioned on the first few samples to generate a specific audio? Say I want to generate *audio1.wav*, then after training with my dataset, I'd be able to produce that audio given the first N samples.

Thank you for your time

**Comment by fatchord** [comment]
@gcunhase You mean like how you would seed an RNN? Anyway, I would be skeptical - mainly because this is a convolutional model with no hidden state - it doesn't 'remember' anything outside of its receptive field.

**Comment by gcunhase** [comment]
@fatchord Yeah, I mean it like an RNN, like seeding the model with an initiator so it produces a sample that we want.

**Comment by fatchord** [comment]
@gcunhase I don't think it would work without conditioning the model with side information. Once the initial seed moves out of the receptive field  it'll just generate random stuff that it's seen during training.
